### ISSUE TITLE
Timer doesn't appear in installed web apps (Android)

### ISSUE THREAD

**Issue by vanjac** [proginosko/LeechBlockNG#265]
LeechBlock does not seem to work with web apps that have been installed to the home screen in Firefox Android.

Eg. with a block filter set up for github.com:
- Go to github.com in Firefox Android
- Open the menu and choose Install, drop the GitHub icon on your home screen
- Launch the app. The timer will not appear.

Other Firefox add-ons are working fine with these web apps (eg. uBlock origin, Noscript), so I think this is an issue with LeechBlock specifically.

**Comment by proginosko** [comment]
Thanks for this report. I wasn't aware of the web apps feature.

Does LB work at all with web apps? For example, if the site is blocked by time periods rather than time limits.

**Comment by vanjac** [comment]
Yes, actually the time period blocks seem to be working. It's only the timer that's broken. It's not counting down in the background either, if I spend a while in a web app then open the same site in a normal browser, the timer has not changed.

**Comment by proginosko** [comment]
I don't know how the web apps connect with normal browser tabs. I would be interest to know if the time limit option works for web apps, even if the timer isn't shown. For example, set a time limit of 1 minute and see if the site is blocked in the web app after 1 minute.

**Comment by vanjac** [comment]
I just tried this, the time limit doesn't have any effect. I used the web app for more than a minute without any block, and I was still able to open the same page in Firefox with the timer showing 1 minute remaining.

**Comment by vanjac** [comment]
However if I let the timer expire in a regular Firefox tab, then the web app will be blocked.
